src/Symfony/Component/DependencyInjection/Exception/InvalidArgumentException.php

```diff
@@ -11,11 +11,13 @@
 
 namespace Symfony\Component\DependencyInjection\Exception;
 
+use Psr\Container\ContainerException;
+
 /**
  * Base InvalidArgumentException for Dependency Injection component.
  *
  * @author Bulat Shakirzyanov <bulat@theopenskyproject.com>
  */
-class InvalidArgumentException extends \InvalidArgumentException implements ExceptionInterface
+class InvalidArgumentException extends \InvalidArgumentException implements ExceptionInterface, ContainerException
```

Comment:
`EnvNotFound` extends this exception, so no need to add implementation declaration in it.